dts/arm/infineon/xmc4500_F100x1024-pinctrl.dtsi

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022 Schlumberger
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* File autogenerated with python script */
```

Comment:
which python script?

Reply:
> which python script?

it's my own script that I used to parse Infineon headers;
https://pastebin.com/6ED3GKfE

shall I just remove this comment altogether?